### ISSUE TITLE
Simplify position for padded story

### ISSUE THREAD

**Issue by fabiradi** [mertkahyaoglu/storybook-zeplin#26]
When using with padded stories (`parameters.layout: 'padded'`), showing the Zeplin image always requires manual shifting of the image. It would be nice to have it positioned **with the padding** automatically.

The story body has the css class `sb-main-padded sb-show-main`. It could also be possible to "know" in the addon. The following CSS added to [overlay](https://github.com/mertkahyaoglu/storybook-zeplin/blob/master/src/utils/overlay.ts) could be enough (not sure, but it worked when used with chrome's developer tools).

```css
/* storybook-zeplin-preview */
padding: 1rem;
position: relative;
```

> I might be able to implement it myself and create a pull request.

**Comment by yannbf** [comment]
Hey @fabiradi that is a great idea. I'll implement it right away!

**Comment by mertkahyaoglu** [comment]
Fixed via #28. Thanks for the issue @fabiradi.
